storages/backends/gcloud.py

```diff
@@ -20,6 +21,7 @@
 class GoogleCloudFile(File):
     def __init__(self, name, mode, storage):
         self.name = name
+        self.mime_type = mimetypes.guess_type(name)[0]
```

Comment:
When the user should override it? Globally at the Django settings (set the desired MIME type for specific file extension)? Something like:
```
GS_MIME_TYPE = {
    '.css': 'text/css',
    '.mp4': 'video/mp4',
}
```